lib/ui/pages/signup_page.dart

```diff
@@ -0,0 +1,120 @@
+part of 'pages.dart';
+
+class SignupPage extends StatefulWidget {
+  const SignupPage({Key? key}) : super(key: key);
+
+  @override
+  State<SignupPage> createState() => _SignupPageState();
+}
+
+class _SignupPageState extends State<SignupPage> {
+  TapGestureRecognizer _loginButtonTapRecognizer = TapGestureRecognizer();
```

Comment:
same implementation with the login_page.dart
```suggestion
  final TapGestureRecognizer _loginButtonTapRecognizer = TapGestureRecognizer();
```
it should be final. Other suggestions can be seen in the image below.

<img width="793" alt="Screen Shot 2022-12-05 at 16 50 56" src="https://user-images.githubusercontent.com/44092368/205607122-a8a5b624-a8b3-40c9-b953-dcd909de2904.png">